benchmarks/media-streaming/dataset/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cloudsuitetest/debian:base-os
+FROM cloudsuite/debian:base-os
 MAINTAINER Alexandros Daglis <alexandros.daglis@epfl.ch>
```

Comment:
I think this is a problem with almost all the images.
I don't know but it might be a good idea to just remove that altogether as we do not have dedicated maintainers for each benchmark anymore.

Reply:
OK, so I'll remove all maintainers.

Reply:
I skimmed through them quickly and it's split between those with maintainers (e.g., search, serving) and those without (in-memory analytics, Cassandra). 

I agree we should remove all.